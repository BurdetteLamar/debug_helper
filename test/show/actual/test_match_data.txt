---
MatchData (message='test_match_data'):
  size: 3
  regexp: !ruby/regexp /(?<a>.)(?<b>.)/
  captures:
  - '0'
  - '1'
  names:
  - a
  - b
