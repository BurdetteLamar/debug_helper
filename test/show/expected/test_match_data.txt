---
MatchData (message='test_match_data'):
  size: 3
  regexp: !ruby/regexp /(?<a>.)(?<b>.)/
  to_a:
  - '01'
  - '0'
  - '1'
  captures:
  - '0'
  - '1'
  names:
  - a
  - b
